kitten-app/src/scenes/createCat.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import { useForm } from "../stateHooks/useForm";
+
+import createCatServices from "../services/createCatServices";
+function CreateCat() {
+    // defining the initial state for the form
+    const initialState = {
+        name: "",
+    };
+
+    // getting the event handlers from our custom hook
+    const { onChange, onSubmit } = useForm(
+        createCatServices,
```

Comment:
[question] If we pass `createCatService` as first parameter, what will actually happen?